lib/measures/default_feature_reports/measure.rb

```diff
@@ -203,6 +242,682 @@ def run(runner, user_arguments)
     # DLM: measures like the GeoJSON to OSM measure can set this value
     feature_report.program.footprint_area = convert_units(floor_area, 'm^2', 'ft^2')
 
+    # maximum_number_of_stories
+    number_of_stories = building.standardsNumberOfStories.get if building.standardsNumberOfStories.is_initialized
+    number_of_stories ||= 1
```

Comment:
I'm not sure if this is a good default value, might be better to just leave results blank if we don't have a good guess or way to calculate